zingolib/src/wallet/notes/transparent.rs

```diff
@@ -199,10 +199,10 @@ pub mod mocks {
     use crate::{test_framework::mocks::build_method, wallet::notes::TransparentOutput};
 
     /// to create a mock TransparentOutput
-    pub struct TransparentOutputBuilder {
+    pub(crate) struct TransparentOutputBuilder {
         address: Option<String>,
         txid: Option<TxId>,
-        output_index: Option<u64>,
+        pub output_index: Option<u64>,
```

Comment:
Should this be an `Option<u32>`?

Reply:
this should stay the same in the builder, and soon zingolib should catch up